vrecord

```diff
@@ -37,6 +37,25 @@ MPVOPTS+=(--load-scripts=no)
 MPVOPTS+=(--script "${SCRIPTDIR}/qcview.lua")
 MPVOPTS+=(--really-quiet)
 
+# Set variables for system differences
+SYSTEM_TEST="$(uname -s)"
+if [[ "${SYSTEM_TEST}" = "Darwin" ]] ; then
```

Comment:
I think the variable could be better named.

Reply:
would you prefer something more explicit? FIND_SYSTEM_TYPE?